### PR TITLE
[importer] add host in ENV

### PR DESCRIPTION
**Pull request by prhod**
To ease the deployment on a different infra, there is the need to add this ENV variable.
In the `.env` file, the `POSTGRES_HOST` var is already present, no other change seems to be necessary

### CODE REVIEW

File: importer/import.py [hunk 1]
```diff
@@ -20,11 +20,12 @@ def retry_if_db_error(exception):
 # we wait a bit for postgres to be ready since if used with docker, so docker might take a while to start
 @retry(stop_max_delay=10000, retry_on_exception=retry_if_db_error)
 def _pg_connect():
+    host = environ.get("POSTGRES_HOST")
```
Comment (antoine-de) [inline importer/import.py:23]: for this to be retrocompatible (but should we?), maybe you can add a default value `postgres` ?

**Review by antoine-de — APPROVED** [review]

File: importer/import.py (hunk 1 above)
Comment (prhod) [reply importer/import.py:23]: Yes, I wasn't sure about this one. The readme indicates that docker and docker-compose are a prerequisit, and the `.env` file is stated in the docker-compose...
I don't think it's necessary, but if you insists I can add it. What do you think ?

Comment (amatissart) [reply importer/import.py:23]: Yes the `POSTGRES_HOST` is already present in the default `.env` and used by the API, so it should be ok.

**Review by amatissart — APPROVED** [review]

File: importer/import.py (hunk 1 above)
Comment (antoine-de) [reply importer/import.py:23]: top!

**Comment by prhod** [comment]
thanks :)

**Pull request merged**